.github/ISSUE_TEMPLATE/page-request.yml

```
@@ -0,0 +1,35 @@
+name: 📄 Page request
+title: "page request: "
+description: Page request template
+labels: new command
+body:
+  - type: textarea
+    attributes:
+      label: Command description
+      description: Describe a command you want to be summarized
+      placeholder: Tell us what Tl;Dr page you want to see!
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Command details
+      description: Describe any details related to a command
+      placeholder: Tell us a command version for instance
+  - type: dropdown
+    attributes:
+      label: OS
+      description: Specify an operation system this command is implemented for (if you don't know please select "Other")
```

```suggestion
      description: Specify the operating system(s) this command is implemented for
```

If a user is requesting a command, they surely know what OS it works on.